tests/transport/rtps_direct_write/Domain.cpp

```diff
@@ -45,11 +45,11 @@ Domain::Domain(int argc, ACE_TCHAR* argv[])
   } catch (const CORBA::BAD_PARAM& e) {
     e._tao_print_exception("ERROR: CORBA::BAD_PARAM");
     cleanup();
-    throw e;
+    throw;
   } catch (const OpenDDS::DCPS::Transport::NotConfigured& e) {
     ACE_ERROR((LM_ERROR, "ERROR: OpenDDS::DCPS::Transport::NotConfigured.\n"));
     cleanup();
-    throw e;
+    throw;
```

Comment:
I didn't know you could rethrow an exception like that.

Reply:
Yeah, it can be required when you catch using an ellipsis (...) but don't
want to prevent the exception from escalating or don't want to
inadvertently change from a base class to an (unknown to current context)
derived exception type.

Reply:
You can *only* rethrow like that